_docs/request-matching.md

```diff
@@ -97,6 +99,32 @@ JSON:
     }
 }
 ```
+## Request with Form Parameters
+
+```java
+stubFor(post(urlPathEqualTo("/mock"))
+.withFormParam("tool", equalTo("WireMock"))
+.withBasicAuth("jeff@example.com", "jeffteenjefftyjeff")
+)
+.willReturn(aResponse()));
+```
+
+```json
+{
+    "request": {
+        "urlPath": "/mock",
+        "method": "POST",
+      "formParameters": {
+            "tool": {
+                "equalTo": "WireMock"
+            }
+        },
+    "response": {
+        "status": 200
+    }
+}
+```
+
 
 The following sections describe each type of matching strategy in detail.
```

Comment:
Are you sure it is placed correctly? AFAICT it is not aligned with the order of the list above